src/ngraph/runtime/cpu/cpu_external_function.cpp

```diff
@@ -685,9 +685,16 @@ using namespace ngraph::runtime;
     writer << "mkldnn_scratchpad_mds = std::vector<mkldnn::memory::desc*>("
            << to_string(m_mkldnn_emitter->get_mkldnn_scratchpad_mds().size()) << ");\n";
     writer << "size_t scratchpad_size = " << m_mkldnn_emitter->get_max_scratchpad_size() << ";\n";
+    writer << "if (scratchpad_size > 0)\n";
+    writer.block_begin();
```

Comment:
Do we need these nested writer.block_begin()  / end() ?

Reply:
yes, it writes "{}" and  adds indent.